bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -58,6 +58,9 @@
  */
 @NonNullByDefault
 public class ExecHandler extends BaseThingHandler {
+
+    private static final String EXEC_HANDLER_THREADPOOL_NAME = "execBinding";
```

Comment:
I’m not sure this is according to the naming convention. Unfortunately i could not find the issue/pr with that convention again on my phone. Unless @lolodomo knows it have to wait till Monday.

Reply:
I just added a comment about that. I don't think we have a naming convention for thread pool. But what is sure is that @Nadahar introduced a new one.

Reply:
I guess so the convention was probably for individual threads. Must say that i don’t disagree with this convention. Only thing that comes to mind is that binding vs addon, but i don’t care that much as either convention perfectly fits in the cfg file.

Reply:
Let’s continue in the other comment